### PR TITLE
Conventions and basic setup is done

### PR DESCRIPTION
**Pull request by awaisdelta**
- Basic architecture is setup
- Components conventions are setup
- `craco` is integrated to reduce directory path issues
- `styled-components integrated for styling convention

### CODE REVIEW

File: craco.config.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+// craco.config.js
+const path = require(`path`);
+const alias = require(`./src/config/aliases`);
+
+const SRC = `./src`;
```
Comment (ShooterArk) [inline craco.config.js:5]: Please use one terminology. change 'SRC' to 'src'

File: src/components/Banner.tsx [hunk 2]
```diff
@@ -1,25 +1,28 @@
 import React from 'react'
-import { Button } from 'reactstrap';
+import Button from './ui/button/Button'
+import { Button as Btn} from 'reactstrap';
```
Comment (ShooterArk) [inline src/components/Banner.tsx:3]: Use One library for the component, either use custom button or reactstrap. please DO NOT use two library for the same component.

File: src/components/Banner.tsx [hunk 3]
```diff
@@ -1,25 +1,28 @@
...
-            </header>
+  return (
+    <div>
+      <header className="masthead home-background mb-5">
+        <div className="container h-100">
+          <div className="row">
+            <div className="col-lg-6 col-sm-12 mx-auto my-auto">
+              <h1 className="text-white mt-sm-25">Contrax Wallet</h1>
+              <p className="text-white"> The Contrax web app simplifies the process of developing, testing, deploying and managing smart contracts with an intuitive interface. </p>
+              {/* <Button label={'Connect Wallet'} variant="light" /> */}
```
Comment (ShooterArk) [inline src/components/Banner.tsx:15]: remove commented code.

File: src/utils/routes.js [hunk 4]
```diff
@@ -0,0 +1,5 @@
+export const pages = {
```
Comment (ShooterArk) [inline src/utils/routes.js:1]: please rename pages to Routes.

**Review by ShooterArk — CHANGES_REQUESTED** [review]

File: src/components/Banner.tsx (hunk 2 above)
Comment (awaisdelta) [reply src/components/Banner.tsx]: done

File: craco.config.js (hunk 1 above)
Comment (awaisdelta) [reply craco.config.js]: done

**Review by ShooterArk — APPROVED** [review]
Initial structure is ready, needs more improvements but that can be handled during project as well.

**Pull request merged**